systemtest/src/test/java/io/strimzi/systemtest/kafka/dynamicconfiguration/DynamicConfST.java

```diff
@@ -224,8 +226,9 @@ void testUpdateToExternalListenerCausesRollingRestart() {
 
         updateAndVerifyDynConf(Environment.TEST_SUITE_NAMESPACE, testStorage.getClusterName(), deepCopyOfShardKafkaConfig);
 
-        kafkaConfigurationFromPod = KafkaCmdClient.describeKafkaBrokerUsingPodCli(Environment.TEST_SUITE_NAMESPACE, scraperPodName, KafkaResources.plainBootstrapAddress(testStorage.getClusterName()), podNum);
-        assertThat(kafkaConfigurationFromPod, containsString("compression.type=snappy"));
+        // Wait until the configuration is properly set and returned by Kafka Admin API
+        StUtils.waitUntilSupplierIsSatisfied(() ->
```

Comment:
Will this produce some relevant message in the wait method? Or will it be `Wait until true is satisfied`? Applies to other places.

Reply:
Updated

Reply:
still applies.